### PR TITLE
docs(helpers): fix a slight error

### PR DESCRIPTION
**Pull request by Fgaoxing**
## Check List

**Please read and check followings before submitting a PR.**

- [ ] I want to publish my theme on Hexo official website.
  - [ ] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
  - [ ] `name` is unique.
  - [ ] `link` URL is correct.
  - [ ] `preview` URL is correct.
  - [ ] `preview` URL web site is rendered correctly.
  - [ ] Add a screenshot to `source/themes/screenshots`.
  - [ ] Screenshot filename is same as value of `name`.
  - [ ] Screenshot size is `800 * 500`.
  - [ ] Screenshot file format is `png`.
- [ ] I want to publish my plugin on Hexo official website.
  - [ ] I have read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
  - [ ] `name` is unique.
  - [ ] `link` URL is correct.
- [x] Others (Update, fix, translation, etc...)
  - Languages:
  - [x] `en` English
  - [x] `ru` Russian
  - [x] `zh-cn` simplified Chinese
  - [x] `zh-tw` traditional Chinese

> **Note**
>
> No need to translate `ko`, `pt-br`, `th` languages if you submit a PR for documentation.

> **Warning**
>
> `ko`, `pt-br`, `th` languages document are inactive. Please, submit issue if you want to continuous contribution to them.

### CODE REVIEW

File: source/docs/helpers.md [hunk 1]
```diff
@@ -272,7 +272,7 @@ Option | Description | Default
 // <link rel="alternate" href="/atom.xml" title="Hexo" type="application/atom+xml">
 
 <%- feed_tag('rss.xml', { title: 'RSS Feed', type: 'rss' }) %>
-// <link rel="alternate" href="/atom.xml" title="RSS Feed" type="application/atom+xml">
+// <link rel="alternate" href="/rss.xml" title="RSS Feed" type="application/atom+xml">
```
Comment (stevenjoezhang) [inline source/docs/helpers.md:275]: Should it be `type="application/atom+xml"` or `type="application/rss+xml"`?

Comment (Fgaoxing) [reply source/docs/helpers.md:275]: 的确，一会改
Indeed, it will be changed later

Comment (Fgaoxing) [reply source/docs/helpers.md:275]: 但具体要看这个helper的实际内容

**Review by tomap — APPROVED** [review]

**Comment by Fgaoxing** [comment]
可以给我通过吗？

**Comment by Fgaoxing** [comment]
@tomap

**Comment by Fgaoxing** [comment]
@tomap Est - il possible de passer ma soumission, s'il vous plaît? May I ask if I can submit it? 请问可以通过我的提交吗？

File: source/docs/helpers.md (hunk 1 above)
Comment (uiolee) [inline source/docs/helpers.md:275]:
```suggestion
// <link rel="alternate" href="/rss.xml" title="RSS Feed" type="application/rss+xml">
```

**Pull request closed without merging**